conf/proxy.yaml

```diff
@@ -4,7 +4,7 @@ proxy:
   tcp-keep-alive: true
   max-connections: 1000
   pd-addrs: "127.0.0.1:2379"
-  # require-backend-tls: true
+  require-backend-tls: false
```

Comment:
You can change code https://github.com/pingcap/TiProxy/blob/main/lib/config/proxy.go#L119 instead.

Reply:
Weird, I reverted it before you reviewed it...